src/tutorials/0005-regular-files-api/07.js

```diff
@@ -7,23 +7,26 @@ const validate = async (result, ipfs) => {
     }
   }
 
-  if (result.error) {
-    if (result.error.toString().includes('file does not exist')) {
+  if (result instanceof Error) {
+    if (result.code === utils.ipfs.errorCodes.ERR_NOT_FOUND) {
       return {
-        fail: "Oops, we could not find a file with that IPFS path. Are you sure you're using the correct path with the correct CID? Remember, if you use the wrapping directory's CID, you need to append `/fun/success.txt` to the path name. Otherwise, if you're using the `fun` subdirectory's CID', you need to append `/success.txt`."
+        fail: "Oops, we could not find a file with that IPFS path. Are you sure you're using the correct path with the correct CID? Remember, if you use the wrapping directory's CID, you need to append `/fun/success.txt` to the path name. Otherwise, if you're using the `fun` subdirectory's CID', you need to append `/success.txt`.",
```

Comment:
```suggestion
        fail: "Oops, we could not find a file with that IPFS path. Are you sure you're using the correct path with the correct CID? Try checking for typos. Remember, if you use the wrapping directory's CID, you need to append `/fun/success.txt` to the path name. Otherwise, if you're using the `fun` subdirectory's CID, you need to append `/success.txt`.",
```